dripline/extensions/modbus/ethernet_modbus_service.py

```diff
@@ -0,0 +1,160 @@
+try:
+    import pymodbus
+    from pymodbus.client import ModbusTcpClient
+    from pymodbus.payload import BinaryPayloadDecoder

```

Comment:
Is no longer available in pymodbus version 3.11.4. I do not see explicitly to which version is used here.